blueprints/data-solutions/gcs-to-bq-with-least-privileges/variables.tf

```diff
@@ -19,18 +19,19 @@ variable "cmek_encryption" {
 }
 
 variable "data_eng_principals" {
-  description = "Groups with Service Account Token creator role on service accounts in IAM format, eg 'group:group@domain.com'."
+  description = "Groups with Service Account Token creator role on service accounts, eg 'group@domain.com'."
```

Comment:
I guess now this principals will have wider permissions, might be worth documenting here. Also, I think that they need to have a `group:` prefix now as they have moved back to `iam` / `iam_additive`